src/schema/v2/fair_organizer.ts

```diff
@@ -23,6 +27,69 @@ export const FairOrganizerType = new GraphQLObjectType<any, ResolverContext>({
       about: {
         type: GraphQLString,
       },
+      articlesConnection: {
+        description: "A connection of articles related to a partner.",
+        type: articleConnection.connectionType,
+        args: pageable({
+          sort: ArticleSorts,
+          page: { type: GraphQLInt },
+          inEditorialFeed: {
+            type: GraphQLBoolean,
+            description:
+              "Articles that are ready to be publicly viewed in the feed by everyone.",
+          },
+        }),
+        resolve: async (
+          { profile_id },
+          args: {
+            inEditorialFeed?: Boolean
+            sort?: ArticleSort
+          } & CursorPageable,
+          { articlesLoader, fairsLoader }
+        ) => {
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const gravityOptions = {
+            fair_organizer_id: profile_id,
+            size: 100,
+          }
+
+          const { body } = await fairsLoader(gravityOptions)
+
+          interface ArticleArgs {
+            published: boolean
+            fair_ids: [string]
+            limit: number
+            count: boolean
+            offset: number
+            sort?: ArticleSort
+            in_editorial_feed?: Boolean
+          }
+
+          const articleArgs: ArticleArgs = {
+            published: true,
+            fair_ids: body.map((fair) => fair._id),
+            limit: size,
+            count: true,
+            offset,
+            sort: args.sort,
+            in_editorial_feed: args.inEditorialFeed,
+          }
+
+          const { results, count } = await articlesLoader(articleArgs)
+
+          return {
+            totalCount: count,
+            pageCursors: createPageCursors({ ...args, page, size }, count),
+            ...connectionFromArraySlice(results, args, {
+              arrayLength: count,
+              sliceStart: offset,
+            }),
+          }
+        },
+      },
```

Comment:
@dblandin - This stuff is understandably copy-paste at this point but would still be nice if we could follow-up with some test coverage

Reply:
>  would still be nice if we could follow-up with some test coverage

Absolutely 👍🏼  I'll get some in tomorrow. Wanted to get these changes out in the limited time I had today to unblock @anastasiapyzhik's work on the new fair organizer page so was a bit aggressive on the commit/push/merge.

Reply:
Np! And thanks for the follow-up 🙏

Reply:
Follow-up PR here: https://github.com/artsy/metaphysics/pull/3322